### PR TITLE
deliveroo-local: use local-only Dockerfile

### PR DESCRIPTION
**Pull request by jeffreylo**
This patch generates a Dockerfile that includes the deliveroo-local-ca certs.

### CODE REVIEW

**Comment by jeffreylo** [comment]
@humzashah: this is for local-use only! It unblocks a few things that we’re working on internally. Ideally, I think we’d have a publicly available base image to use, but them’s the breaks…

File: .deliveroo-local/make/generate-dockerfile.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+RUN apt-get update && apt-get install -y sudo && rm -rf /var/lib/apt/lists/*
+
+# Copy the certs from the deliveroo-local-ca image.
+COPY --from=dev-certs \
+     /usr/share/ca-certificates/deliveroo-local-ca.crt \
+     /usr/share/ca-certificates/deliveroo-local-ca.crt
+
+# Add the ca-cert to the store.
+RUN echo "deliveroo-local-ca.crt" >> /etc/ca-certificates.conf && \
+    sudo update-ca-certificates --fresh
```
Comment (humzashah) [inline .deliveroo-local/make/generate-dockerfile.sh]: is `sudo` necessary here?

**Review by humzashah — APPROVED** [review]
added a minor non-blocking comment but LGTM

File: .deliveroo-local/make/generate-dockerfile.sh (hunk 1 above)
Comment (jeffreylo) [reply .deliveroo-local/make/generate-dockerfile.sh]: Doesn’t seem like it! Removed this and the `sudo` installation in 2c552ee. Good catch! ❤

**Pull request merged**